htdocs/manage/settings/index.bml

```diff
@@ -445,6 +466,23 @@ body<=
     $ret .= $cats_with_settings{$given_cat}->{desc};
     $ret .= "</p></div>";
 
+    $ret .= "<div class='settings_content'>";
+    $ret .= "<div class='$given_cat'>";
+
+    if ( $given_cat eq "notifications" && $can_view_other ) {
+        # form for viewing another user's notifications
```

Comment:
:+1: No sense delaying an immediate need for the "grand amazingness of the future".